lib/services/parameter-store/stores/ssm/get-batch-parameters.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+const { getSsmClient } = require('./get-ssm-client');
+const get = require('lodash.get');
+const chunk = require('lodash.chunk');
+const flatten = require('lodash.flatten');
+
+const getParametersFromSSM = ({ parameterNames }) => {
+  const ssm = getSsmClient();
+
+  return ssm.getParameters({
+    Names: parameterNames,
+    WithDecryption: true
+  })
+  .promise()
+  .then(res => {
+    const validParameters = get(res, 'Parameters') || [];
+    const invalidParameters = get(res, 'InvalidParameters') || [];
```

Comment:
Is `lodash.get` useful here given you're not getting a nested key + you've already done a fallback if undefined via `||`? 🤔

Reply:
I never trust JS :P. If `res` is not an object then it would throw.
Waiting for the day to do `res.InvalidParameters?`